src/keytar/win.rs

```diff
@@ -1,43 +1,108 @@
 use super::error::KeytarError;
 use std::ffi::c_void;
 use std::result::Result;
-use windows::{core::*, Win32::Foundation::*, Win32::Security::Credentials::*};
+use windows_sys::{
+  core::{PCWSTR, PWSTR},
+  Win32::Foundation::*,
+  Win32::Security::Credentials::*,
+  Win32::System::{
+    Diagnostics::Debug::{
+      FormatMessageW, FORMAT_MESSAGE_ALLOCATE_BUFFER, FORMAT_MESSAGE_FROM_SYSTEM,
+      FORMAT_MESSAGE_IGNORE_INSERTS,
+    },
+    Memory::LocalFree,
+  },
+};
 
 impl From<WIN32_ERROR> for KeytarError {
   fn from(error: WIN32_ERROR) -> Self {
-    KeytarError::Os(error.to_hresult().message().to_string())
+    KeytarError::Os(win32_error_as_string(error))
   }
 }
 
+/**
+ * Helper function to convert the last Win32 error into a human-readable error message.
+ * Returns: A String containing the error message.
+ */
+fn win32_error_as_string(error: WIN32_ERROR) -> String {
+  let buffer: PWSTR = std::ptr::null_mut();
+
+  // https://github.com/microsoft/windows-rs/blob/master/crates/libs/core/src/hresult.rs#L96
```

Comment:
unfortunately we can't since `from_win32` is not exposed outside of the crate 😔 

I can try pulling it in and comparing local file sizes, but it may increase as a result.